curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-form-element.md

```diff
@@ -11,7 +11,7 @@ You can build web forms that actually submit data to a server using nothing more
 
 For example:
 
-`<form action="/url-where-you-want-to-submit-form-data"></form>`
+`<form action="/url-where-you-want-to-submit-form-data"><input></form>`
```

Comment:
````suggestion
```html
<form action="/url-where-you-want-to-submit-form-data">
  <input>
</form>
```
````

Reply:
Thanks, yeah that's definitely better. Just committed that change.